database/indexPG.js

```diff
@@ -0,0 +1,61 @@
+const {Pool} = require('pg');
+
+const pool = new Pool({
+  user: 'postgres',
+  host: 'localhost',
+  database: 'reviews',
+  password: 'hackreactor',
+  port: 5432,
+});
+// client.connect();
+
+const getMessages = (listing_id, callback) => {
+  let sql = `SELECT * FROM bnbReview LEFT JOIN bnbUsers ON bnbReview.user_id = bnbUsers.id WHERE listing_id = ${listing_id}`;
```

Comment:
Consider refactoring your query scripts to use parameterized variables, protecting your DB from SQL injection attacks.